### PR TITLE
Add custom diagram colors to badges; refine initial-load documentation

### PR DESCRIPTION
**Pull request by gpdf**

### CODE REVIEW

File: pipeline-module/README.md [hunk 1]
```diff
@@ -90,32 +99,35 @@ Check on the GitHub site for the repository to make sure that the initial “Act
...
 See the [spherex-tex](https://github.com/SPHEREx/spherex-tex) repository's [README](https://github.com/SPHEREx/spherex-tex/blob/main/README.md) for instructions.
 
-Once ready, push this content to GitHub:
+f. Once ready, push this content to GitHub:
 
 ```shell
 git add .
 git commit
-git push -u
+git push -u origin initial-text
```
Comment (gpdf) [inline pipeline-module/README.md:130]: @jonathansick I don't know if bare `git push -u` does the right thing if one has certain config options set, but for us it doesn't seem to work.

File: pipeline-module/README.md [hunk 2]
```diff
@@ -44,6 +44,15 @@ git add .
 git commit -m "Initialize repo"
 ```
 
+NB: If your version of `git` is too old to support the `git -b` option, you can do the following instead:
```
Comment (gpdf) [inline pipeline-module/README.md:47]: This is the case for folks who are taking the Apple default version of `git` from macOS Catalina.

File: pipeline-module/{{cookiecutter.handle|lower|slugify}}/README.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # {{ cookiecutter.handle }}: {{ cookiecutter.module_name }}
 
-![Pipeline Level: {{ cookiecutter.level }}](https://img.shields.io/badge/Pipeline-{{ cookiecutter.level }}-blue)
-![Diagram Index: {{ cookiecutter.diagram_index }}](https://img.shields.io/badge/Diagram__Index-{{ cookiecutter.diagram_index }}-blue)
+![Pipeline Level: {{ cookiecutter.level }}](https://img.shields.io/badge/Pipeline-{{ cookiecutter.level }}-{% if cookiecutter.level == "L1" %}88BFFE{% elif cookiecutter.level == "L2" %}1AC604{% elif cookiecutter.level == "L3" %}BEB1D0{% else %}FEBCFF{% endif %})
```
Comment (gpdf) [inline pipeline-module/{{cookiecutter.handle|lower|slugify}}/README.md:3]: This is a bit frivolous, but cute.  We have a standard PDF showing all the pipeline modules - it's the diagram referred to by "diagram index" and I thought I'd just color-code the level badges to match the colors used on that diagram, which I just picked up from inspecting the PDF.

File: pipeline-module/README.md (hunk 1 above)
Comment (jonathansick) [reply pipeline-module/README.md:130]: Ah, yes, the configuration to make that work is:

```
[push]
	default = current
```

Going back to your recommended change, I think it should be:

```
git push -u origin main
```

That way the `main` branch of the local repository corresponds to the `main` branch of the GitHub origin.

**Review by jonathansick — APPROVED** [review]

**Pull request merged**